src/id128/id128.c

```diff
@@ -63,6 +66,112 @@ static int verb_invocation_id(int argc, char **argv, void *userdata) {
         return id128_pretty_print(id, arg_mode);
 }
 
+typedef struct gpt_table_entry {
+        const char *name;
+        const char *cname;
+        sd_id128_t uuid;
+} gpt_table_entry;
+
+static gpt_table_entry gpt_table[] = {
+        { "gpt-root-x86",              "GPT_ROOT_X86",              GPT_ROOT_X86    },
+        { "gpt-root-x86-64",           "GPT_ROOT_X86_64",           GPT_ROOT_X86_64 },
+        { "gpt-root-arm   ",           "GPT_ROOT_ARM",              GPT_ROOT_ARM    },
+        { "gpt-root-arm_64",           "GPT_ROOT_ARM_64",           GPT_ROOT_ARM_64 },
```

Comment:
hmm, the underscore is weird

Reply:
Oh, right. I wanted it to be x86_64 and arm-64, but changed the wrong place.